src/optionsStorage.ts

```diff
@@ -55,6 +55,7 @@ const defaultOptions = {
   viewBobbing: true,
   packetsLoggerPreset: 'all' as 'all' | 'no-buffers',
   serversAutoVersionSelect: 'auto' as 'auto' | 'latest' | '1.20.4' | string,
+  customChannels: true,
```

Comment:
```suggestion
  customChannels: false,
```

should not be enabled by default to avoid bans